### PR TITLE
fix issues with semantic release

### PR DESCRIPTION
**Pull request by kantord**
https://github.com/kiwicom/smart-faq/issues/474<br/><br/><br/><url>LiveURL: https://smartfaq-fix-issues-with-semantic-release.surge.sh</url>

### CODE REVIEW

**Comment by jalbertsr** [comment]
maybe you should update DOCs on how it would work now, last time commitlint failed https://circleci.com/gh/kiwicom/smart-faq/4995

**Comment by kantord** [comment]
@jalbertsr I added docs here https://github.com/kiwicom/smart-faq/pull/607/commits/8c8817daa4ee9af0d6e26ae7218df7fd86cb7bab

Do you think some more information is needed?

**Comment by kantord** [comment]
fixed the commitlint issues too

**Comment by kantord** [comment]
@jaroslav-kubicek no, actually GitHub releases with release notes should be created semantic-release.

For example here all of my release notes were generated by semantic-release:
https://github.com/kantord/just-dashboard/releases/tag/v0.8.0

**Comment by jaroslav-kubicek** [comment]
cool, that looks great, the no. of releases will just increase, so we will have usually multiple release notes like this for one MR, right? it also means we should disable squashing after this is merge...

**Comment by kantord** [comment]
> we will have usually multiple release notes like this for one MR, right?

One CI run will always generate one release notes. Each line in the release notes corresponds to one commit. Which means that most of the time one Pull Request will produce one release, since we usually merge to master directly. Essentially, we'll release everything immediately after merge.

> it also means we should disable squashing after this is merge...

Maybe, but not necessarily. I mean we should still use squash as a method of producing more user-friendly commit messages. For example if we have "feat: implement card title" and "feat: implement card perex" we could squash them into "feat: implement card". The only thing we **have to** watch out for is that the squashed commits have to obey the format as well, and have to be user-friendly. To avoid mistakes with that, commits will be validated after commit, and also before release.

File: .circleci/config.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ version: 2
 defaults: &defaults
   working_directory: ~/smart-faq
   docker:
-    - image: circleci/node:8.0
+    - image: circleci/node:9.0
```
Comment (jalbertsr) [inline .circleci/config.yml]: why this change? is it strictly necessary? I think the LTS version is `8.11.3`

**Comment by kantord** [comment]
rebased @jalbertsr @jaroslav-kubicek

**Comment by kantord** [comment]
rebased again @jalbertsr :roll_eyes:

**Comment by kantord** [comment]
Merge?
